### PR TITLE
US: fix ESPN source

### LINKED ISSUE #3225: Fix: US ESPN 1
US ESPN 1 has quit opening..

**_Channel Name:_** ESPN 1
**_Broken Link (from playlist):_** https://gma2.blab.email/fs2.m3u8
**_Possible Replacement (optional):_** N/a
**_Notes (optional):_** xxx

### PR DESCRIPTION
**Pull request by Dum4G**
Closes https://github.com/iptv-org/iptv/issues/3225

### CODE REVIEW

File: channels/us.m3u [hunk 1]
```diff
@@ -1486,13 +1486,13 @@ https://zypelive-lh.akamaihd.net/i/default_1@710948/master.m3u8
...
 http://k3.usastreams.com:1935/etvSD/etvSD/playlist.m3u8
 #EXTINF:-1 tvg-id="ESPN.us" tvg-name="ESPN" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Sports",ESPN (720p)
-https://gma2.blab.email/fs2.m3u8
+http://164.68.124.111/fs2.m3u8
 #EXTINF:-1 tvg-id="ESPN2.us" tvg-name="ESPN 2" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Sports",ESPN 2 (720p)
-https://gma2.blab.email/espn2.m3u8
+http://164.68.124.111/espn2.m3u8
 #EXTINF:-1 tvg-id="ESPNNews.us" tvg-name="ESPN News" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Sports",ESPN News (720p)
-https://gma2.blab.email/espnews.m3u8
+http://164.68.124.111/espnews.m3u8
```
Comment (LaneSh4d0w) [inline channels/us.m3u:1493]: Doesn't work.

Comment (Dum4G) [reply channels/us.m3u:1493]: File technicaly exists and downloads. Let's test if it's not 24/7

Comment (LaneSh4d0w) [reply channels/us.m3u:1493]: Re-verified, still a 0 octet file being sent.

Comment (Dum4G) [reply channels/us.m3u:1493]: Will remove it tomorrow if nothing changes

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**